<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGeneralPlan</class>
 <widget class="QDialog" name="DialogGeneralPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Размеры участка</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QComboBox" name="CB_Top_Neighbour"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Расстояние до красной линии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SB_RedLineDistanceTop"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QComboBox" name="CB_Left_Neighbour"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Расстояние до красной линии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SB_RedLineDistanceLeft"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QComboBox" name="CB_Right_Neighbour"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Расстояние до красной линии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SB_RedLineDistanceRight"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QComboBox" name="CB_Bottom_Neighbour"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Расстояние до красной линии</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SB_RedLineDistanceBottom"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Размеры участка</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Длина</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SB_Width">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ширина</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SB_Height">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGeneralPlan</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGeneralPlan</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
